.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
```

Comment:
```suggestion
    rev: v4.1.0
```

Reply:
I enabled dependabot for this repo, so it should take care of opening PRs for verison updates

Reply:
Dependabot doesn't support `pre-commit` https://github.com/dependabot/dependabot-core/issues/1524

Reply:
Oh woops, I thought this was a github actions file (not sure why). Is there something else I can use to update all the versions locally, or do I have to go through each one manually?

Reply:
I found `pre-commit autoupdate` 😄